recipes/ssht/all/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class SshtTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package", "cmake_paths"
```

Comment:
Typically we use https://github.com/conan-io/conan-center-index/blob/eb5f141d056d92020e3bee7bb4f9eeca52633a3f/recipes/any-lite/all/test_package/conanfile.py#L7

```suggestion
    generators = "cmake", "cmake_find_package"
```